### DISCUSSION TITLE
Water Resistance? (solo2 in a wristband?)

### DISCUSSION THREAD

**Discussion by colemickens** [solokeys/solo2#120] [General]
How water resistant is the Solo2?

I really want to commission a custom wristband that would allow to conceal a Solo2 (and separately my building's access fob) with maybe some sort of magnetic USB connector exposed. I could then tap my wrist to my phone for NFC operations, and/or use it tethered to my computer via the magnetic usb cable (and worst case scenario, I guess potentially destroy the wristband to retrieve the original-form device.

Obviously, water would be a bit of a concern here. I'm not sure how the Solo2 will handle potentially being "soaked" in the wristband if it gets wet, I'd be concerned about the magnetic connector, but I know I've washed my Yubikeys before and they've come out the other side. (Also, not that it's a fair comparison given the size of the efforts behind them but the Yubikeys strike me as a bit more tolerant of abuse than the Solo2 keys.)

Any thoughts? Anyone else stupidly obsessed with the idea of "wearing" their security key? Maybe you have some fun ideas, too?

**Comment by needs-coffee** [comment]
pretty much all PCBs are fine with getting wet its only when you plug them in and they are wet you have a problem. Salty water (i.e. sweat) may present an issue however if it is "soaked" long term with corrosion, especially the Type-c version as the rest of the contacts on the PCB are ENIG but the type-c connector is off the shelf of unknown material. 

Cant testify to the durability of the solokeys as i cant really use mine at the moment but the yubikeys can take a beating - have had a 5c on my keys for close to 4 years and its got no issues.

Whats wrong with putting it on your keys? Presumably you need keys?

**Comment by colemickens** [comment]
To set the right tone, this is some nerdy dorky business, it's nothing terribly serious. I have had a 'yubikey 4+4c' attached to my keyring for years, as well. I can also attest that they take quite a beating, even the longer/thinner full-form usb-a variant.

If I weren't aiming for "the ideal", I'd just solder/glue some sort of magnetic connector to a usb-a yubikey and, with backups in hand, cross my fingers. However, I am still holding out hope that Solo2 will get PIV support soon and I can replace the yubikey(s) with some good ole rust/oss.

>Whats wrong with putting it on your keys? Presumably you need keys?

There are *some* times where I'm places that I'd prefer to not have to have my keys or keep track of them. And sadly, despite my attempts at being methodical, I seem to keep losing things. I hope that would be harder to do if said item were strapped to my wrist. My normal keys include my mailbox key, physical-fallback door key, a quickish-style detach ring, etc -- not something I would want to strap to my body.

[There is one more possibly even **sillier** reason to have a wrist attached device...](https://github.com/NateBrune/silk-guardian)
